cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -364,6 +364,8 @@ data:
     <match kubernetes.**>
       @type google_cloud
 
+      # Try to detect JSON formatted log entries.
+      detect_json true
       # Collect metrics in Prometheus registry about plugin activity.
       enable_monitoring true
       monitoring_type prometheus
```

Comment:
Is this a good time to sync with the official config and make the buffer_chunk_size smaller (we have it at 1M)? Smaller bundles have less latency, allowing for more throughput with fewer threads... Also below.

Reply:
Done, thanks!